### ISSUE TITLE
Understanding the loss and accuracy outputs

### ISSUE THREAD

**Issue by DarylWM** [tflearn/tflearn#105]
I'm training an LSTM to categorise time series data, and I'm not understanding the learning trends I'm seeing in the logs.
![image](https://cloud.githubusercontent.com/assets/1016303/15491619/99941972-21b5-11e6-8742-583cd23043ea.png)
![image](https://cloud.githubusercontent.com/assets/1016303/15491626/b0524288-21b5-11e6-94f6-3af78265b18d.png)
Any idea why the total accuracy is increasing quite nicely, but the validation accuracy is decreasing? Maybe I don't understand the difference. I thought 'total accuracy' meant the final accuracy of the trained model, and 'validation accuracy' meant the accuracy of the model against the validation set for that training step. Am I close?

**Comment by aymericdamien** [comment]
It seems your network is overfitting, you may try some ops such as batch_normalization or dropout to avoid such issues.

**Comment by rahulchandnani** [comment]
@DarylWM  I'm experiencing the same problem like validation loss increases after few epochs (in time series binary classification).
Did you find the solution ?
Could you please share like where did you apply batch normalization layer ?

Regards
Rahul Chandnani
